types/chart.js/index.d.ts

```diff
@@ -52,6 +53,8 @@ declare class Chart {
     getElementsAtXAxis: (e: any) => Array<{}>;
     getDatasetAtEvent: (e: any) => Array<{}>;
     getDatasetMeta: (index: number) => Meta;
+    isDatasetVisible: (datasetIndex: number) => boolean;
+    setDatasetVisibility: (datasetIndex: number, visible: boolean) => void;
```

Comment:
Why stop there? We're also missing `get..`. and `toggle...`.

Reply:
@lekoaf: quite a lot more are missing and not just for this class :/
I wanted my first contribution to be small (and manageable as a side task - took me long enough to figure out how to properly do the testing - none of the existing tests for this package had used any '// $Expect...' so far)
I added the other visibility related methods now but they are somewhat strange and I have not used them in practice.

Reply:
What does `// $Expect` do?

Reply:
> What does `// $Expect` do?

I was asking myself exactly the same

Reply:
nothing it turns out :/ I didn't do a negative check and simply assumed it was like another test framework I used. I am somewhat surprised that the tests do not appear to actually test what is begin given back but only what the typings say - why do the heavy lifting of executing chart.js in a DOM when the types are not verified?

Reply:
> nothing it turns out :/ I didn't do a negative check and simply assumed it was like another test framework I used. I am somewhat surprised that the tests do not appear to actually test what is begin given back but only what the typings say - why do the heavy lifting of executing chart.js in a DOM when the types are not verified?

... I re-read the documentation that the test is not actually run, my bad

Reply:
> Why stop there? We're also missing `get..`. and `toggle...`.
I had another look, the members you mentioned are not actually part of chart.js@2.9.x - they are in chart.js 3 that is still in "alpha/pre-release" stage, so I removed them again
* https://github.com/chartjs/Chart.js/blob/2.9/src/core/core.controller.js
* https://github.com/chartjs/Chart.js/blob/release-3.x/src/core/core.controller.js

Reply:
What? They have alpha/pre-release stuff on the master branch?? Hmm, oh well.